docs/source/installation/developer-anaconda.rst

```diff
@@ -43,4 +45,63 @@ Quick start
 Detailed instructions
 ---------------------
 
```

Comment:
Might be good to add a comment saying that the detailed instructions give the explanations on how the quick start instructions work, but the commands are the same. Maybe something like
```suggestion
The following is a detailed explanation of the steps provided in the Quick start section above.
Refer to that section for the full text of the various commands.

```

Reply:
Agreed. I've also gone ahead and added it to the pip developer installation too, since that is where I copy-pasted a lot of this from anyway.